### ISSUE TITLE
[pre-5.0] Unable to start scenario with 2 nodes using Python API

### ISSUE THREAD

**Issue by justas-** [coreemu/core#135]
My Python CORE scenario crashes when using current "master" branch and trying to add two nodes to simulation.

The relevant part of my CORE Python script:

```
def main():
    main_start_time = time.time()

    # Init the session
    global session
    session = Session(random.randint(1, 9), persistent = True)
    global server
    try:
        server.addsession(session)
    except:
        print('No Server in globals()')

    load_services()

    # Build routers
    router1 = session.add_object(cls=nodes.CoreNode, name='router1')
    router1.setposition(x=200, y=200)
    session.services.addservicestonode(router1, 'router', 'IPForward|zebra|OSPFv2')

    router2 = session.add_object(cls=nodes.CoreNode, name='router2')
    router2.setposition(x=400, y=200)
    session.services.addservicestonode(router2, 'router', 'IPForward|zebra|OSPFv2')
```

The runtime output:

> Traceback (most recent call last):
>   File "lbe-dumbell.py", line 254, in <module>
>     main()
>   File "lbe-dumbell.py", line 139, in main
>     router2 = session.add_object(cls=nodes.CoreNode, name='router2')
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/session.py", line 589, in add_object
>     obj = cls(self, *clsargs, **clskwds)
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/netns/vnode.py", line 564, in __init__
>     self.startup()
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/netns/vnode.py", line 591, in startup
>     super(LxcNode, self).startup()
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/netns/vnode.py", line 81, in startup
>     env = self.session.get_environment(state=False)
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/session.py", line 509, in get_environment
>     env["SESSION_NODE_COUNT"] = "%s" % self.get_node_count()
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/session.py", line 842, in get_node_count
>     self.objects))
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/session.py", line 841, in <lambda>
>     count = len(filter(lambda x: not nodeutils.is_node(x, (NodeTypes.PEER_TO_PEER, NodeTypes.CONTROL_NET)),
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/misc/nodeutils.py", line 60, in is_node
>     type_class = get_node_class(node_type)
>   File "/usr/local/lib/python2.7/dist-packages/core_python-5.0-py2.7.egg/core/misc/nodeutils.py", line 45, in get_node_class
>     return _NODE_MAP[node_type]
> TypeError: 'NoneType' object has no attribute '__getitem__'
> Closing session

**Comment by bharnden** [comment]
There was a change in how to configure nodes, which required as extra step. I will probably look to do a better pass in general, but this should be fixed in 5.1 going forward as there will be default nodes configured for a session.

**Comment by bharnden** [comment]
Nodes are now set by default, this avoids this issue
